deps/obs-scripting/obs-scripting-python.c

```diff
@@ -1654,9 +1654,7 @@ bool obs_scripting_load_python(const char *python_path)
 	/* ---------------------------------------------- */
 	/* Load main interface module                     */
 
-	char *absolute_script_path = os_get_abs_path_ptr(SCRIPT_DIR);
```

Comment:
Removing this part will make Python support break on windows.

Reply:
The whole purpose of the changes to the libobs is to allow setting the paths of the resources that will be used at runtime. If you know a better way of doing it, please let me know.

Reply:
> Removing this part will make Python support break on windows.

Could you provide us with more insight on how this will break on Windows?